src/qibolab/platform.py

```diff
@@ -352,6 +352,123 @@ def create_tii_rfsoc4x2(runcard, address=None):
     return platform
 
 
+def create_tii_zcu111(runcard, address=None):
+    """Create platform using QICK project on the ZCU111 board and EraSynth local oscillator for the Readout
+    IPs and other instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+    Args:
+        runcard (str): Path to the runcard file.
+        address (str): Address and port for the QICK board.
+            If ``None`` it will attempt to connect to TII instruments.
+    """
+    # from qibolab.instruments.dummy_oscillator import (
+    #    DummyLocalOscillator as LocalOscillator,
+    # )
+    from qibolab.instruments.erasynth import ERA
+    from qibolab.instruments.rfsoc import RFSoC as TII_ZCU111
+
+    # Create channel objects
+    channels = ChannelMap()
+
+    # QUBIT 0 # D2
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
```

Comment:
After #449 (and a fix, both already in `main`) you can just write:
```suggestion
    channels |= "L3-30_ro"  # dac6
```